Wikipedia/Code/WMFAppViewController.m

```diff
@@ -1417,7 +1417,7 @@ - (void)setNotificationsCenterButtonForExploreViewController:(ExploreViewControl
         NSInteger numUnreadNotifications = [[self.dataStore.remoteNotificationsController numberOfUnreadNotificationsAndReturnError:nil] integerValue];
         UIImage *image = numUnreadNotifications == 0 ? [self notificationsCenterBellImageWithUnreadNotifications:NO] : [self notificationsCenterBellImageWithUnreadNotifications:YES];
         UIBarButtonItem *notificationsBarButton = [[UIBarButtonItem alloc] initWithImage:image style:UIBarButtonItemStylePlain target:_exploreViewController action:@selector(userDidTapNotificationsCenter)];
-        notificationsBarButton.accessibilityLabel = [WMFCommonStrings notificationsCenterTitle];
+        notificationsBarButton.accessibilityLabel = numUnreadNotifications == 0 ? [WMFCommonStrings notificationsCenterTitle] : [WMFCommonStrings notificationsCenterBadgeTitle];
```

Comment:
We'll need this same logic in `WMFSettingsViewController.m` too - currently it doesn't switch to the `notificationsCenterBadgeTitle` type when it shows the badged icon.

Reply:
Great catch! I forgot to test with the explore feed off